src/engine/BMInterfaceGame.php

```diff
@@ -49,23 +48,43 @@ public function create_game(
         }
 
         try {
-            $gameId = $this->insert_new_game($playerIdArray, $maxWins, $description, $previousGameId);
+            if (!isset($currentPlayerId)) {
+                if ($this->isTest) {
```

Comment:
I hear you, but if you're changing the interface, you have to change the tests.

One thing i do a lot in responderTest is to create a wrapper around an API call, if there's a common way i always want to call that API.  Then i only have to implement to API changes in one place.